backend/sites/src/app/services/landHistory/landHistory.service.ts

```diff
@@ -0,0 +1,18 @@
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { LandHistories } from 'src/app/entities/landHistories.entity';
+
+export class LandHistoryService {
+    constructor(
+        @InjectRepository(LandHistories)
+        private landHistoryRepository: Repository<LandHistories>,
+    ) { }
+
+    async getLandHistoriesForSite(siteId: string): Promise<LandHistories[]> {
+        try {
+            return this.landHistoryRepository.find({ where: { siteId } });
+        } catch (error) {
+            throw error;
+        }
+    }
+}
```

Comment:
pls include unit tests for resolver and services

Reply:
That's the plan! I didn't expect you to review a Draft PR 😄

Reply:
the team raises the PR when the task is done, not before that 😄